static/app/utils/theme.tsx

```diff
@@ -1101,8 +1102,6 @@ const commonTheme = {
 
     cardTitle: {
       fontSize: '1rem',
```

Comment:
Any reason we still need this?

Reply:
I dont think so, I must have forgotten to remove it, good catch!